recipes/nd2reader/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "nd2reader" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "18c721250669de804b85b7295e9885c94c21f99918b8da451ebbbab84fe29612" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.6.2
+    - six >=1.4
+    - xmltodict >=0.9.2
+    - pims >=0.3.0
+  run:
+    - python
+    - numpy >=1.6.2
+    - six >=1.4
+    - xmltodict >=0.9.2
+    - pims >=0.3.0
+
+test:
+  imports:
+    - nd2reader
+
+about:
+  home: https://github.com/rbnvrw/nd2reader
+  license: GPL-3.0
+  summary: 'Pure Python library for reading NIS Elements ND2 images and metadata'
+
+  description: |
+    nd2reader is a pure-Python package that reads images produced by NIS Elements 4.0+. It has 
+    only been definitively tested on NIS Elements 4.30.02 Build 1053. Support for older versions 
+    is being actively worked on. The reader is written in the pims framework, enabling easy 
+    access to multidimensional files, lazy slicing, and nice display in IPython.
+
+extra:
+  recipe-maintainers:
+    - rbnvrw
```

Comment:
@rbnvrw - are you ok with being added as a co-maintainer of the nd2reader package on conda-forge?